users.json

```diff
@@ -0,0 +1,1423 @@
+{
```

Comment:
we have to decide what is our 'externalId'

that could be the fiscal number (which is not unique) or the SPID id (unique but there could be more than one for each user)